client/signup/steps/user/index.jsx

```diff
@@ -73,7 +74,17 @@ export class UserStep extends Component {
 	}
 
 	componentWillMount() {
+		const { oauth2Signup, initialContext } = this.props,
+			clientId =
+				get( initialContext, 'query.oauth2_client_id', null )
```

Comment:
=) Sorry, me again

there is no need for the ternary operator, `get` either gets the variable you requested or returns `null`, so it's all handled, sorry my previous comment wasn't clear enough

Reply:
No, that's me being daft and not reading my own code properly. Thanks for pointing that out!